model/src/main/kotlin/Project.kt

```diff
@@ -48,6 +51,13 @@ data class Project(
          */
         val declaredLicenses: SortedSet<String>,
 
+        /**
+         * The declared licenses as [SpdxExpression]. If [declaredLicenses] contains multiple licenses they are
+         * concatenated with [SpdxOperator.AND].
+         */
+        @JsonInclude(JsonInclude.Include.NON_NULL)
+        val declaredLicensesProcessed: SpdxExpression?,
```

Comment:
Instead of adding very similar code to all package manager, should we maybe just assign a default value bas on processing `declaredLicenses` to `declaredLicensesProcessed`?

Reply:
I first wanted to do this, but could not find a way to implement error handling correctly, as errors for unmapped licenses need to be added to the analyzer result.

Reply:
I have changed the field to be an object that contains the expression and the unmapped licenses, this made it possible to initialize the field in the constructor and remove all the code from the package manager classes.